### PR TITLE
[WIP]Russian translation

### PR DESCRIPTION
**Pull request by Anonym0uz**
Hi, i'd like to help with translation into Russian language :)

- [x] menu_darwin.cson
- [x] menu_win32.cson
- [x] menu_linux.cson
- [x] context.cson
- [ ] settings.cson
- [x] about.cson

### CODE REVIEW

**Comment by liuderchi** [comment]
@Anonym0uz Thank you for contributing :100: 
Now you can push your commits to update this PR in any time.

I'll leave this PR open and please notify me if you want to merge it.

File: def/ru/menu_darwin.cson [hunk 1]
```diff
@@ -280,31 +280,31 @@ Menu:
...
+    value: "Окно"
     submenu:
       Minimize:
-        value: "Minimize"
+        value: "Свернуть"
       Zoom:
-        value: "Zoom"
-      "Bring All to Front":
+        value: "Расширить"
+      "Все окна - на передний план":
```
Comment (liuderchi) [inline def/ru/menu_darwin.cson]: Line no.299 `"Bring All to Front":` is the key so please keep it unchanged.

Comment (Anonym0uz) [inline def/ru/menu_darwin.cson]: Hmm, i already changed it recently... Strange.

File: def/ru/menu_darwin.cson [hunk 2]
```diff
@@ -296,24 +302,24 @@ Menu:
         value: "Свернуть"
       Zoom:
         value: "Расширить"
-      "Все окна - на передний план":
-        value: "Bring All to Front"
+      "Bring All to Front":
+        value: "Все окна - на передний план"
```
Comment (Anonym0uz) [inline def/ru/menu_darwin.cson]: Yep, changed :)

File: def/ru/menu_darwin.cson [hunk 3]
```diff
@@ -3,317 +3,323 @@ Menu:
     value: "Atom"
     submenu:
       "About Atom":
-        value: "About Atom"
+        value: "О Atom"
       "View License":
-        value: "View License"
+        value: "Лицензия"
+      "Check For Update":
+        value: "Проверить обновления"
```
Comment (liuderchi) [inline def/ru/menu_darwin.cson]: Thanks for pointing out new item 

`"Check For Update"` to be translated. 

Confirmed this shows up in win32 platform also (not in linux).
May you help to add this to `menu_win32.cson` also? :smile:

Comment (Anonym0uz) [inline def/ru/menu_darwin.cson]: Oh yeah, i'll add for win32 file, in next hours i update translation :)

**Comment by 4LL1M1xuXkHtoHafW2DObp9raIQR5JkpNLo2Cun** [comment]
So what waiting?

**Comment by liuderchi** [comment]
There is still one missing entry in `menu_win32.cson`
Will fix it myself after merging it.

**Comment by Anonym0uz** [comment]
Added "Check for update" in menu_win32.cson file and 75-80% translated settings.cson. Soon all settings translate.

**Comment by liuderchi** [comment]
@Anonym0uz Sorry about mering this branch without notifying,

Please keep updating your translation (whether on this branch or new one is fine)
and I'll try to merge rest of it.

**Pull request merged**